.github/workflows/build-test-containers.yaml

```
@@ -0,0 +1,55 @@
+name: Build Test Containers
+run-name: Build Test Containers
+on: [push]
```

Could we add the option to manually trigger them? e.g. if we might want to re-gen them to check the e2e process is still working? Also we might only want to run this automatically if a related file changes if we end up using this repo for other thing?
```suggestion
on:
  workflow_dispatch:
  push:
    branches:
      - 'main'
    paths:
      - 'container-images'
      - '.github/workflows/build-test-containers.yaml'
```